src/content/doc-surrealdb/querying/graphql/http.mdx

```diff
@@ -34,10 +34,23 @@ $env:SURREAL_EXPERIMENTAL_GRAPHQL=true surreal start --log debug --user root --p
 </TabItem>
 </Tabs>
 
+In order to allow querying the created table using GraphQL, you will need to explicitly enable graphql using the [`DEFINE CONFIG`](/docs/surrealql/statements/define/config) statement. This will allow you to query the table using GraphQL on a per-database basis.
```

Comment:
```suggestion
In order to allow querying the created table using GraphQL, you will need to explicitly enable GraphQL using the [`DEFINE CONFIG`](/docs/surrealql/statements/define/config) statement. This will allow you to query the table using GraphQL on a per-database basis.
```